src/Stryker.Core/Stryker.Core.UnitTest/StringWriterExtenstions.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Stryker.Core.UnitTest
+{
+    internal static class TextWriterExtenstions
+    {
+        private const string Escape = "\u001b";
+
+        private static readonly Regex Ansi = new Regex("\u001b\\[[\\d;]+m", RegexOptions.ECMAScript);
+
+        /// <summary>
+        /// Removes ANSI escape sequences from the <paramref name="stringWriter"/>.
+        /// </summary>
+        /// <returns>A basic string without ANSI escape sequences.</returns>
+        public static string RemoveAnsi(this TextWriter stringWriter)
+        {
+            return Ansi.Replace(stringWriter.ToString(), string.Empty);
+        }
+
+        /// <summary>
+        /// Counts the number of green spans in the <paramref name="stringWriter"/>.
+        /// </summary>
+        /// <returns>The number of green spans.</returns>
+        public static int GreenSpanCount(this TextWriter stringWriter)
```

Comment:
Very nice! Looks much better in our tests :)

Reply:
Glad you like it :)